lms/validation/_helpers/_base.py

```diff
@@ -22,6 +22,23 @@ class Meta:
 class PyramidRequestSchema(_BaseSchema):
     """Base class for schemas that validate Pyramid requests."""
 
+    locations = None
+    """
+    The locations where webargs should look for the parameters.
+
+    If this is ``None`` then webargs's default locations will be used.
+
+    Subclasses can override this to control where parameters are searched for::
+
+        class MySchema(PyramidRequestSchema):
+            locations = ["form"]
+
+            ...
+
+    For the list of available locations see:
+    https://webargs.readthedocs.io/en/latest/quickstart.html#request-locations
```

Comment:
Can you think of a scenario where we might want to use the same schema in different contexts but search in different locations?

Reply:
I'm not sure whether that'll come up or not. Currently you would have to subclass the schema and override its `locations` attr. If the need comes up in future we can devise a way to specify locations in view config. The standard webargs view decorator (which we aren't using because it's a bad fit for Pyramid) lets you do just that: https://webargs.readthedocs.io/en/latest/framework_support.html#id6